lookbook/test/components/previews/beta/auto_complete_preview.rb

```diff
@@ -3,38 +3,292 @@
+        c.leading_visual_icon(icon: :search)
+        c.results do
+          render(Primer::Beta::AutoComplete::Item.new(disabled: true, value: "apple")) do |_c|
+            Apple
+          end
+          render(Primer::Beta::AutoComplete::Item.new(value: "orange")) do |_c|
+            Orange
+          end
+        end
+      end
     end
 
     # @!endgroup
+
+    # @label Leading visual in results
+    # @param label_text text
+    # @param show_clear_button toggle
+    # @param visually_hide_label toggle
+    # @param placeholder text
+    # @param size select [small, medium, large]
+    # @param full_width toggle
+    # @param disabled toggle
+    # @param invalid toggle
+    # @param input_id text
+    # @param list_id text
+    # @param input_name text
+    def leading_visual_in_results(label_text: "Select a fruit", show_clear_button: false, visually_hide_label: false, placeholder: "Placeholder text", size: :medium, full_width: false, disabled: false, invalid: false, input_id: "input-id", list_id: "list-id", input_name: "input-id")
+      render(Primer::Beta::AutoComplete.new(label_text: label_text, input_id: input_id, list_id: list_id, src: "/auto_complete", show_clear_button: show_clear_button, visually_hide_label: visually_hide_label, placeholder: placeholder, size: size, full_width: full_width, disabled: disabled, invalid: invalid, input_name: input_name)) do |c|
+        c.results do
+          render(Primer::Beta::AutoComplete::Item.new(value: "apple")) do |_c|
+            c.leading_visual_icon(icon: :info)
+            Apple
+          end
+          render(Primer::Beta::AutoComplete::Item.new(value: "orange")) do |_c|
+            Orange
+          end
+        end
+      end
+    end
+
+    # @label Trailing visual in results
+    # @param label_text text
+    # @param show_clear_button toggle
+    # @param visually_hide_label toggle
+    # @param placeholder text
+    # @param size select [small, medium, large]
+    # @param full_width toggle
+    # @param disabled toggle
+    # @param invalid toggle
+    # @param input_id text
+    # @param list_id text
+    # @param input_name text
+    def trailing_visual_in_results(label_text: "Select a fruit", show_clear_button: false, visually_hide_label: false, placeholder: "Placeholder text", size: :medium, full_width: false, disabled: false, invalid: false, input_id: "input-id", list_id: "list-id", input_name: "input-id")
+      render(Primer::Beta::AutoComplete.new(label_text: label_text, input_id: input_id, list_id: list_id, src: "/auto_complete", show_clear_button: show_clear_button, visually_hide_label: visually_hide_label, placeholder: placeholder, size: size, full_width: full_width, disabled: disabled, invalid: invalid, input_name: input_name)) do |c|
+        c.results do
+          render(Primer::Beta::AutoComplete::Item.new(value: "apple")) do |_c|
+            c.trailing_visual_icon(icon: :info)
```

Comment:
@joelhawksley I couldn't figure out how to make these leading/trailing visuals render. It seems like maybe there's JS somewhere populating these test Autocomplete results.. any ideas?

Reply:
This is another case of needing to put this in a preview template like we did the other day ❤️  (Since we can only call `render` once)

Reply:
For clarity, we discovered results just.. doesn't really do anything. You _must_ use the `src` param to pass in `items`.